packages/util/src/Percentage.ts

```diff
@@ -0,0 +1,48 @@
+import { BigIntMath } from './BigIntMath';
+
+/**
+ * Calculates the rounded percentages for each part from {@link total}.
+ * When total is omitted, it is assumed that the total is the sum of the parts.
+ *
+ * Dividing bigints is always rounded down, creating this problem:
+ * sum([...percentage(part)]) != percentage(sum([...parts]))
+ * This function will be distribute the difference to the highest percentage value.
```

Comment:
Consider changing the strategy here: for an edge case where only **some** coins are delegated I would rather see 1% than 0%.
```suggestion
 * This function will be distribute the difference to the lowest percentage value.
```

Reply:
No longer valid. Percentages are now floating point numbers between 0 and 1, of type `Percent`